tests/group_list_connectors_mock_test.go

```diff
@@ -0,0 +1,113 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/fivetran/go-fivetran"
+	"github.com/fivetran/go-fivetran/tests/mock"
+)
+
+func TestGroupListConnectorsServiceDo(t *testing.T) {
+	// arrange
+	groupID := "projected_sickle"
+	limit := 10
+	cursor := "eyJza2lwIjoxfQ"
+	schema := "salesforce"
+
+	ftClient, mockClient := CreateTestClient()
+	handler := mockClient.When(http.MethodGet, fmt.Sprintf("/v1/groups/%s/connectors", groupID)).
+		ThenCall(func(req *http.Request) (*http.Response, error) {
+			response := mock.NewResponse(req, http.StatusOK, prepareGroupListConnectorsResponse())
+			return response, nil
+		})
+
+	// act
+	response, err := ftClient.NewGroupListConnectors().
+		GroupID(groupID).
+		Limit(limit).
+		Cursor(cursor).
+		Schema(schema).
+		Do(context.Background())
+
+	// assert
+	if err != nil {
+		t.Error(err)
+	}
+
+	interactions := mockClient.Interactions()
+	assertEqual(t, len(interactions), 1)
+	assertEqual(t, interactions[0].Handler, handler)
+	assertEqual(t, handler.Interactions, 1)
+	assertGroupListConnectorsResponse(t, response)
+}
+
+func prepareGroupListConnectorsResponse() string {
+	return `{
+		"code": "Success",
+		"data": {
+			"items": [
+				{
+					"id": "iodize_impressive",
+					"group_id": "projected_sickle",
+					"service": "salesforce",
+					"service_version": 1,
+					"schema": "salesforce",
+					"connected_by": "concerning_batch",
+					"created_at": "2018-07-21T22:55:21.724201Z",
+					"succeeded_at": "2018-12-26T17:58:18.245Z",
+					"failed_at": "2018-08-24T15:24:58.872491Z",
+					"sync_frequency": 60,
+					"status": {
+						"setup_state": "connected",
+						"sync_state": "paused",
+						"update_state": "delayed",
+						"is_historical_sync": false,
+						"tasks": [],
+						"warnings": []
+					}
+				}
+			],
+			"next_cursor": "eyJza2lwIjoxfQ"
+		}
+	}`
+}
+
+func assertGroupListConnectorsResponse(t *testing.T, response fivetran.GroupListConnectorsResponse) {
+	isHistoricalSyncFlag := false
+	createdAt, _ := time.Parse(time.RFC3339, "2018-07-21T22:55:21.724201Z")
+	succeededAt, _ := time.Parse(time.RFC3339, "2018-12-26T17:58:18.245Z")
+	failedAt, _ := time.Parse(time.RFC3339, "2018-08-24T15:24:58.872491Z")
```

Comment:
Define just string constants, convert values from response and compare strings inside assertions. 
You could define `assertTimeEqual(t *t.Testing, actual time.Time, string expected)`
And convert passed expected value inside this func.